app/Models/Customer.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Customer extends Model
+{
+    use HasFactory;
+
+    public function getFullNameAttribute()
+    {
+        return "{$this->first_name} {$this->last_name}";
+    }
```

Comment:
This is called an Accessor in Laravel, which can be used to create new, computed values from existing attributes.

https://laravel.com/docs/8.x/eloquent-mutators#defining-an-accessor